components/resource_control/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use online_config::OnlineConfig;
+use serde::{Deserialize, Serialize};
+
+mod resource_group;
+pub use resource_group::{ResourceController, ResourceGroupManager, MIN_PRIORITY_UPDATE_INTERVAL};
+
+mod future;
+pub use future::ControlledFuture;
+
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug, OnlineConfig, Default)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    #[online_config(skip)]
+    pub enabled: bool,
```

Comment:
please also add a sample in config-tempalte.toml

Reply:
@Connor1996 PTAL, do you feel ok with this new config

Reply:
looks good

Reply:
Bad name. Should be `enable`.